src/server.rs

```diff
@@ -103,6 +103,48 @@ impl StreamInner {
     }
 }
 
+#[derive(Copy, Clone)]
+struct CleanupState {
```

Comment:
non blocking: would be cool to track all the session variables the user has set and print them :)

Reply:
The command tag (server response) is how we determine if there was a set statement or not. It does not include the actual variable that was set unfortunately so we'd have to parse the SET query and then do a few other things there to get it to log. Not trivial to add that in this PR, but I'd love to have that visibility